cloud-openstack/src/main/java/com/sequenceiq/cloudbreak/cloud/openstack/HeatTemplateBuilder.java

```diff
@@ -0,0 +1,82 @@
+package com.sequenceiq.cloudbreak.cloud.openstack;
+
+import static org.springframework.ui.freemarker.FreeMarkerTemplateUtils.processTemplateIntoString;
+
+import java.io.IOException;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import javax.inject.Inject;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Service;
+
+import com.sequenceiq.cloudbreak.cloud.model.CloudCredential;
+import com.sequenceiq.cloudbreak.cloud.model.Group;
+import com.sequenceiq.cloudbreak.cloud.model.Image;
+import com.sequenceiq.cloudbreak.cloud.model.Network;
+import com.sequenceiq.cloudbreak.cloud.model.Security;
+import com.sequenceiq.cloudbreak.cloud.openstack.view.KeystoneCredentialView;
+import com.sequenceiq.cloudbreak.cloud.openstack.view.NeutronNetworkView;
+import com.sequenceiq.cloudbreak.cloud.openstack.view.NovaInstanceView;
+import com.sequenceiq.cloudbreak.cloud.openstack.view.OpenStackGroupView;
+import com.sequenceiq.cloudbreak.domain.InstanceGroupType;
+
+import freemarker.template.Configuration;
+import freemarker.template.TemplateException;
+
+@Service
+public class HeatTemplateBuilder {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(HeatTemplateBuilder.class);
+
+    @Value("${cb.openstack.heat.template.path:templates/openstack-heat.ftl}")
```

Comment:
It's a different module and has nothing to do with the core, so no. Although it might be a good idea to collect these values in a dedicated class, but within the module.